docs/cel2/notices/pectra-upgrade.md

```diff
@@ -2,12 +2,12 @@
 
 This page outlines breaking changes related to the Ethereum Pectra (Prague-Electra) L1 hard fork for node operators on the Celo Alfajores L2 testnet.
 
-See also the [Optimism Pectra update information](https://docs.optimism.io/builders/notices/pectra-changes).
+See also the [Optimism Pectra update information](https://docs.optimism.io/notices/pectra-changes).
 
 :::info
-This page will be updated continuously with information on upgrade runbooks and timelines as they come. This based on the last Ethereum [ACDE on Jan 30th 2025](https://ethereum-magicians.org/t/all-core-devs-execution-acde-204-january-30-2025/22594). These dates are subject to change based on how `devnet-6` goes. Ethereum L1 clients are currently scheduled to have a release out by Feb 10th.
+This page will be updated continuously with information on upgrade runbooks and timelines as they come. Ethereum L1 clients are currently scheduled to have a release by Feb 13th.
 
-The Pectra upgrade for Holesky L1 proposed activation is on Wed, Feb 19, 2025.
+The Pectra upgrade for Holesky L1 is proposed for slot: 3710976 (Mon, Feb 24 at 21:55:12 UTC).
```

Comment:
Where are you getting this information ?

Reply:
https://discord.com/channels/1244729134312198194/1331301843829002313/1337071297745915956